pkg/registration/hub/manager.go

```diff
@@ -76,6 +78,8 @@ func (m *HubManagerOptions) AddFlags(fs *pflag.FlagSet) {
 			"The flag works only when ResourceCleanup feature gate is enable.")
 	fs.StringVar(&m.HubClusterArn, "hub-cluster-arn", m.HubClusterArn,
 		"Hub Cluster Arn required to connect to Hub and create IAM Roles and Policies")
+	fs.StringSliceVar(&m.AutoApprovedCSRUsers, "auto-approved-csr-users", m.AutoApprovedCSRUsers, "")
+	fs.StringSliceVar(&m.AutoApprovedARNPatterns, "auto-approved-arn-patterns", m.AutoApprovedARNPatterns, "")
```

Comment:
please add description for the flag `auto-approved-csr-users` and `auto-approved-arn-pattern`

And for flag `cluster-auto-approval-users`, we should mark it deprecated, because the `auto-approved-csr-users` will be instead of `cluster-auto-approval-users`

Reply:
Like this?

```fs.MarkDeprecated("cluster-auto-approval-users", "use auto-approved-csr-users flag instead of this")```

Reply:
addressed